tf_environment.yml

```diff
@@ -0,0 +1,8 @@
+name: pyvertical-dev-tf
+channels:
+  - tensorflow
```

Comment:
remove the `tensorflow` channel and add `conda-forge`, else conda won't be able to find numpy or pandas

Reply:
Fixed.